test/functional/plugin/lsp/testutil.lua

```diff
@@ -39,7 +39,8 @@ M.create_server_definition = [[
         })
         local handler = handlers[method]
         if handler then
-          handler(method, params, callback)
+          local response, err = handler(method, params)
```

Comment:
Not sure why but this change is because https://github.com/neovim/neovim/pull/28914 seems to have broken my test logic. If this change was intended though I'll modify the tests.

Reply:
Never mind, I understand the change now and I will remove this change from the PR.